app/src/test/java/org/mozilla/fenix/components/TrackingProtectionPolicyFactoryTest.kt

```diff
@@ -91,34 +91,6 @@ class TrackingProtectionPolicyFactoryTest {
         expected.assertPolicyEquals(always, checkPrivacy = false)
     }
 
-    @Test
-    fun `cookiePurging must be available ONLY in nightly or debug`() {
-        mockkObject(Config)
-        for (channel in ReleaseChannel.values()) {
-            every { Config.channel } returns channel
-
-            val shouldCookiePurgingActive = channel.isNightlyOrDebug
-            val customSetting =
-                settingsForCustom(shouldBlockCookiesInCustom = true, blockCookiesSelection = "all")
-            val stringSetting = mockSettings(useStrict = true)
-            val recommendedSetting = mockSettings(useTrackingProtection = true)
-
-            for (setting in arrayOf(recommendedSetting, stringSetting, customSetting)) {
-                val factory = TrackingProtectionPolicyFactory(setting)
-                val privateOnly =
-                    factory.createTrackingProtectionPolicy(normalMode = false, privateMode = true)
-                val normalOnly =
-                    factory.createTrackingProtectionPolicy(normalMode = true, privateMode = false)
-                val always =
-                    factory.createTrackingProtectionPolicy(normalMode = true, privateMode = true)
-
-                assertEquals(shouldCookiePurgingActive, privateOnly.cookiePurging)
-                assertEquals(shouldCookiePurgingActive, normalOnly.cookiePurging)
-                assertEquals(shouldCookiePurgingActive, always.cookiePurging)
-            }
-        }
-    }
-
```

Comment:
I think we still want the feature only in nightly or debug?

Reply:
Is there a particular reason why we want to remove the test before allowing in other channels?

Reply:
I added new tests below :)